### ISSUE TITLE
Missing config file in pre-trained Textattack models (CNN, LSTM)

### ISSUE THREAD

**Issue by sanchit97** [QData/TextAttack#485]
**Describe the bug**
Running attacks on Textattack's custom pretrained models - `WordCNNForClassification `and `LSTMForClassification` on AG-News dataset (any dataset with more than 2 classes) is unsuccessful. This is because the file `config.json` is used to initialize model before the state dict is loaded and there is no `config.json` file present in the pre-trained model download from s3. Right now, the default initialization which uses 2 labels is used to initialize the model which results in datasets having 2 classes to work but not datasets like AGNews which have 4 classes.

**To Reproduce**
1. Run following command `python -m textattack attack --model cnn-ag-news --recipe textbugger --num-examples 10`
2. See error `size mismatch for out.bias: copying a param with shape torch.Size([4]) from checkpoint, the shape in current model is torch.Size([2]).`

**Expected behavior**
Should start attack.

**System Information (please complete the following information):**
 - OS: Any
 - Library versions:`torch==1.7.1, transformers==3.3.0`
 - Textattack version - `master`

**Additional context**
Exact place for this behavior to occur is : https://github.com/QData/TextAttack/blob/master/textattack/models/helpers/lstm_for_classification.py#L111

**Comment by sanchit97** [comment]
This was originally pointed out by @qiyanjun. Maybe requires a re-upload of Textattack models or change in `config.json` logic.

**Comment by qiyanjun** [comment]
handling by https://github.com/QData/TextAttack/pull/486

**Comment by jinyongyoo** [comment]
We need to upload separate config.json for each model.
